package/utils/getStyleRule.js

```
@@ -0,0 +1,44 @@
+/* eslint global-require: 0 */
+const { canProcess, moduleExists } = require("./helpers")
+const { requireOrError } = require("./requireOrError")
+const config = require("../config")
+const inliningCss = require("./inliningCss")
```

ESLint

[error] Expected 1 empty line after require statement not followed by another require.

(import/newline-after-import)